.github/workflows/zowe-cli-bundle.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Use Node.js 16
       uses: actions/setup-node@v3
```

Comment:
now that npm 8 comes with node16, should we remove the |Install npm 8 step"?
CC: @t1m0thyj

Reply:
We could try removing it and see what happens. I believe we locked down the npm version to 8.11 because one of the components failed to bundle with 8.12 (https://github.com/zowe/zowe-cli-standalone-package/pull/145) but that issue may have been fixed in newer 8.x versions.

Reply:
sounds good, let's stay at the version that we know works 😋